simulation_parameters/Constants.cs

```diff
@@ -464,6 +464,8 @@ public static class Constants
     /// </summary>
     public const float ENZYME_DIGESTION_EFFICIENCY_MAXIMUM = 0.6f;
 
+    public const float ADDITIONAL_DIGESTIBLE_GLUCOSE_AMOUNT_DENOMINATOR = 4;
```

Comment:
I think this should be a multiplier instead of a divisor, once again because float division is *much* more expensive than multiplying by `0.25f`